api/client/service/opts.go

```diff
@@ -462,7 +463,6 @@ func (opts *serviceOptions) ToService() (swarm.ServiceSpec, error) {
 			Delay:         opts.update.delay,
 			FailureAction: opts.update.onFailure,
 		},
-		Networks:     convertNetworks(opts.networks),
```

Comment:
The whole spec change is designed to to work in a cluster of nodes in different versions. But a docker 1.13 client will always only set the `NetworkAttachmentConfig` in `TaskSpec`.